sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Rollback-drop-student-assessments-index-only-one-assessment-data-null.sql

```diff
@@ -0,0 +1,28 @@
+-- This rollback script is similar to the creation script for the constraint
+-- student_assessments_only_one_assessment_data_null
+-- It is used as the down script for dropping the above constraint
+UPDATE
```

Comment:
Not a blocker at this moment but rollback scripts are not necessarily the copy from the past ones.
This UPDATE would probably be avoided at this moment.